### DISCUSSION TITLE
gitpod help

### DISCUSSION THREAD

**Discussion by Agente3z** [aahnik/tgcf#513] [Q&A]
I cant manage to make it work on gitpod, i run the docker image but it gives me a debug link which opens another editor

**Comment by Lucifer-00007** [comment]
> I cant manage to make it work on gitpod, i run the docker image but it gives me a debug link which opens another editor

Follow these steps:
- Create an empty github repo
- now import the repo to gitpot workspace
- Now run all the terminal commands


```
python3 -m venv .venv

source .venv/bin/activate

pip install tgcf
```
 
 create an .env file and add PASSWORD=your_pass
 
 Now run
 
 `tgcf-web`
 
 Use python 3.11 
 
 @Agente3z  I hope it helped!
